### PR TITLE
Addition of HS60 ISO keyboard

### PR DESCRIPTION
**Pull request by yiancar**

### CODE REVIEW

File: keyboards/hs60/hs60.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#ifndef HS60_H
+#define HS60_H
+
+#define XXX KC_NO
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+
+#define KEYMAP( \
```
Comment (mechmerlin) [inline keyboards/hs60/hs60.h]: rename this to `LAYOUT_60_iso`

Ideally you'll also have a `LAYOUT_60_ansi` defined here.

File: keyboards/hs60/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "hs60.h"
+
+//This is the ISO version of the PCB
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[0] = KEYMAP( /* Base */
```
Comment (mechmerlin) [inline keyboards/hs60/keymaps/default/keymap.c]: rename to `LAYOUT_60_iso`

File: keyboards/hs60/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    KC_GESC,  KC_1,     KC_2,     KC_3,  KC_4,  KC_5,  KC_6,    KC_7,  KC_8,  KC_9,     KC_0,     KC_MINS,  KC_EQL,  KC_BSPC,\
+    KC_TAB,   KC_Q,     KC_W,     KC_E,  KC_R,  KC_T,  KC_Y,    KC_U,  KC_I,  KC_O,     KC_P,     KC_LBRC,  KC_RBRC,         \
+    KC_CAPS,  KC_A,     KC_S,     KC_D,  KC_F,  KC_G,  KC_H,    KC_J,  KC_K,  KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS, KC_ENT, \
+    KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,  KC_C,  KC_V,  KC_B,    KC_N,  KC_M,  KC_COMM,  KC_DOT,   KC_SLSH,           KC_RSFT,\
+    KC_LCTL,  KC_LGUI,  KC_LALT,                       KC_SPC,                          KC_RALT,  MO(1)  ,  KC_APP,  KC_RCTL),
+
+[1] = KEYMAP( /* Base */
```
Comment (mechmerlin) [inline keyboards/hs60/keymaps/default/keymap.c]: rename to `LAYOUT_60_iso`

File: keyboards/hs60/rules.mk [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+RGB_MATRIX_ENABLE = yes     # Use RGB matrix
+
```
Comment (mechmerlin) [inline keyboards/hs60/rules.mk]: add a line to the bottom of this file with

`LAYOUTS = 60_iso`

File: keyboards/hs60/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "hs60.h"
```
Comment (mechmerlin) [inline keyboards/hs60/keymaps/default/keymap.c]: replace this line with `#include QMK_KEYBOARD_H`

Comment (yiancar) [reply keyboards/hs60/keymaps/default/keymap.c]: Done them all thanks for the help !

**Comment by mechmerlin** [comment]
Travis failure is due to having the `60_ansi` in rules.mk but not actually having `LAYOUT_60_ansi` defined in hs60.h

**Comment by yiancar** [comment]
Ok  going to remove user-space layouts for now as some people have configs that dont support rgb and it wont compile correctly.

I dont think there is a good and easy solution for that now.

**Comment by drashna** [comment]
Looks good, thanks!

**Pull request merged**